src/main/java/org/zalando/sprocwrapper/proxy/SProcCallHandler.java

```diff
@@ -41,8 +42,8 @@ private static String getSqlNameForMethod(final String methodName) {
     private RowMapper getRowMapper(SProcCall scA) {
         if (scA.resultMapper() != Void.class) {
             try {
-                return (RowMapper<?>) scA.resultMapper().newInstance();
-            } catch (final InstantiationException | IllegalAccessException ex) {
+                return (RowMapper<?>) scA.resultMapper().getDeclaredConstructor().newInstance();
+            } catch (final InstantiationException | IllegalAccessException | SecurityException | InvocationTargetException | IllegalArgumentException | NoSuchMethodException ex) {
```

Comment:
Could it be replaced with just `Exception` here and in other updated places?

Reply:
i think the work is done now, so lets stick with it.